oem/firstvoices/common/build_keyboards.sh

```diff
@@ -16,17 +14,29 @@ set -u
 # set -x: Debugging use, print each statement
 # set -x
 
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+# . "$(dirname "$THIS_SCRIPT")/../../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$(dirname "$THIS_SCRIPT")/../../../resources/build/jq.inc.sh"
 
+# This build script assumes that the https://github.com/keymanapp/keyboards repo is in
+# the same parent folder as this repo, with the default name 'keyboards'
+KEYBOARDS_ROOT=$(dirname "$THIS_SCRIPT")/../../../keyboards
```

Comment:
```suggestion
KEYBOARDS_ROOT="$KEYMAN_ROOT/../keyboards"
```
(quotes are important, and the path needed to go one level deeper as well!)